File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/ORDERTRANSLayCommonInfo.java
package nta.med.data.model.ihis.nuro;

public class ORDERTRANSLayCommonInfo {
	private String gubunName    ;
	private String ifValidYn   ;
	private String gongbiYn     ;
	public ORDERTRANSLayCommonInfo(String gubunName, String ifValidYn,
			String gongbiYn) {
		super();
		this.gubunName = gubunName;
		this.ifValidYn = ifValidYn;
		this.gongbiYn = gongbiYn;
	}
	public String getGubunName() {
		return gubunName;
	}
	public void setGubunName(String gubunName) {
		this.gubunName = gubunName;
	}
	public String getIfValidYn() {
		return ifValidYn;
	}
	public void setIfValidYn(String ifValidYn) {
		this.ifValidYn = ifValidYn;
	}
	public String getGongbiYn() {
		return gongbiYn;
	}
	public void setGongbiYn(String gongbiYn) {
		this.gongbiYn = gongbiYn;
	}
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/ORDERTRANSLayCommonInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    [Serializable]
    public class ORDERTRANSLayCommonInfo
    {
        private String _gubunName;
        private String _ifValidYn;
        private String _gongbiYn;

        public String GubunName
        {
            get { return this._gubunName; }
            set { this._gubunName = value; }
        }

        public String IfValidYn
        {
            get { return this._ifValidYn; }
            set { this._ifValidYn = value; }
        }

        public String GongbiYn
        {
            get { return this._gongbiYn; }
            set { this._gongbiYn = value; }
        }

        public ORDERTRANSLayCommonInfo() { }

        public ORDERTRANSLayCommonInfo(String gubunName, String ifValidYn, String gongbiYn)
        {
            this._gubunName = gubunName;
            this._ifValidYn = ifValidYn;
            this._gongbiYn = gongbiYn;
        }

    }
}